README.md

```diff
@@ -96,6 +96,20 @@ Put whatever text you want in this static badge, which is independent of a repo.
 
 </div>
 
+### Dynamic data
+
+Create a badge that references any value you want in a given remote JSON file and it will always stay up to date.
```

Comment:
```suggestion
Create a badge that references any value within a public data file (currently only JSON supported). It will always stay up to date.
```